OsmAnd-java/src/main/java/net/osmand/search/core/SearchCoreFactory.java

```diff
@@ -943,6 +945,7 @@ public boolean search(SearchPhrase phrase, SearchResultMatcher resultMatcher) th
 				NameStringMatcher nmAdditional = includeAdditional ?
 						new NameStringMatcher(phrase.getFirstUnknownSearchWord(), StringMatcherMode.CHECK_EQUALS_FROM_SPACE) : null;
 				Map<String, PoiTypeResult> poiTypes = getPoiTypeResults(nm, nmAdditional);
+				poiTypes = filterTypes(poiTypes);
```

Comment:
Before/after 
<img width="300" src="https://github.com/user-attachments/assets/8d7f02ce-605b-43cf-90f5-f0b28135cb41" /> <img width="300" src="https://github.com/user-attachments/assets/6db5278a-43a7-4d9a-abfc-2b3ead9dafda" />

Reply:
Discussed only poi additional part of category should be displayed even then we see strange "non descriptive" wheelchair-yes , so it should be reviewed